package.json

```diff
@@ -434,4 +434,4 @@
     "node": "10.15.2",
     "yarn": "^1.10.1"
   }
-}
```

Comment:
Would you mind installing the `editorconfig` plugin for your editor so that it will avoid inserting newlines where they shouldn't be? https://github.com/elastic/kibana/blob/master/.editorconfig#L12-L13